src/main/kotlin/io/inventi/eventstore/eventhandler/config/EventStoreToolsJdbiConfig.kt

```diff
@@ -3,16 +3,19 @@ package io.inventi.eventstore.eventhandler.config
 import io.inventi.eventstore.eventhandler.annotation.ConditionalOnSubscriptionsEnabled
 import io.inventi.eventstore.eventhandler.dao.ProcessedEventDao
 import io.inventi.eventstore.eventhandler.dao.SubscriptionCheckpointDao
+import io.inventi.eventstore.eventhandler.dao.SubscriptionInitialPositionDao
 import org.jdbi.v3.core.Jdbi
 import org.jdbi.v3.sqlobject.kotlin.onDemand
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.DependsOn
 import org.springframework.jdbc.datasource.TransactionAwareDataSourceProxy
 import javax.sql.DataSource
 
 @ConditionalOnSubscriptionsEnabled
 @Configuration
+@DependsOn(value = ["flyway", "flywayInitializer"])
```

Comment:
I think it is sufficient to add this to `EventStoreToolsSubscriptionsConfiguration`. You also only need `flywayInitializer`

Reply:
Done